edi_oca/models/edi_exchange_record.py

```diff
@@ -603,3 +607,28 @@ def delayable(self, **kw):
 
     def _job_retry_params(self):
         return {}
+
+    def _compute_related_queue_jobs_count(self):
+        for rec in self:
+            rec.related_queue_jobs_count = rec.env["queue.job"].search_count(
+                [("func_string", "like", str(rec))]
```

Comment:
Can we add a TODO? We should refactor the object field on queue_job to use jsonb field so that we can search directy into it.